### PR TITLE
Added instructions for connecting to SFTP To Go

### PR DESCRIPTION
**Pull request by saggineumann**

### CODE REVIEW

File: protocols/profiles/sftptogo.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#### S3
+
+Login to SFTP To Go's dashboard and go to the credentials page:
+
+1. Expand the root credentials to copy the S3 endpoint and secrets.
+
+Enter the following information in the [bookmark](../cyberduck/bookmarks.md):
+
+- Protocol: `Amazon S3`
+- Access Key ID: `Access key ID copied from SFTP To Go's dashboard`
```
Comment (ylangisc) [inline protocols/profiles/sftptogo.md:53]: Missing server part?

Comment (saggineumann) [reply protocols/profiles/sftptogo.md:54]: @ylangisc thank you!  I added server part information

File: protocols/profiles/sftptogo.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#### S3
+
+Login to SFTP To Go's dashboard and go to the credentials page:
+
+1. Expand the root credentials to copy the S3 endpoint and secrets.
+
+Enter the following information in the [bookmark](../cyberduck/bookmarks.md):
+
+- Protocol: `Amazon S3`
+- Server: Copy the bucket and region from SFTP To Go' dashboard and use them in the following pattern: `<bucket>.s3.<region>.amazonaws.com`
```
Comment (dkocher) [inline protocols/profiles/sftptogo.md:53]: This is not recommended to use as an endpoint to connect to a single bucket but according to the [documentation](https://docs.cyberduck.io/protocols/s3/#access-single-bucket) should be `<bucketname>.s3.amazonaws.com` where we will derive the actual endpoint from the bucket location. The actual endpoint then used is the dual stack endpoint `s3.dualstack…` by default..

**Review by dkocher — CHANGES_REQUESTED** [review]

**Comment by dkocher** [comment]
Please rebase to `origin/master` to resolve the build error.

**Pull request closed without merging**